src/client/auth/auth.proto

```diff
@@ -157,17 +157,30 @@ message SetConfigurationRequest {
 }
 message SetConfigurationResponse {}
 
+// AdminGrant reflects any cluster-wide (admin) permissions a principal has.
+// Principals can have multiple admin scopes.
+message AdminGrant {
+  enum Scope {
+    // SUPER gives access to all APIs and owner on everything in PFS (previous just called admin)
+    SUPER = 0;
+
+    // FS gives Owner on all repos in PFS but not access to manage other aspects of the cluster
+    FS = 1;
+  }
+  repeated Scope scopes = 1;
+}
+
 // Get the current list of cluster admins
 message GetAdminsRequest{}
 message GetAdminsResponse{
-  // admins contains the list of cluster admins
-  repeated string admins = 1;
+  // admins contains a mapping of principals to admin grants 
+  map<string, AdminGrant> admins = 1;
 }
 
-// Add or remove cluster admins
+// Set admin grants for the specified admin.
```

Comment:
Similar to above, I think we should mark fields 1 and 2 reserved and make these 3 and 4.

Also, we should make sure there's a way to revoke a user's admin privileges (If we add a `NONE` scope, it could be by setting `grant` to `NONE`, which is how SetScope works)